README.md

```diff
@@ -187,15 +187,17 @@ function Bar (N : Natural) return Natural is
 
 ### If-statements
 
-With one-line condition:
-
 ```Ada
 if <condition> then
    <statements>
+elsif <condition> then
+   <statements>
+else
+   <statements>
 end if;
 ```
 
-With multi-line condition:
+If one of the `<condition>`s does not fit onto one line:
```

Comment:
I'd rephrase that to "If *any* of the `<condition>`s does not fit onto one line:"

Reply:
Fixed by 794c679.